hydromt/stats/skills.py

```diff
@@ -1,21 +1,17 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
+"""Implementation of necessary statistical methods."""
 
-import calendar
 import warnings
-from datetime import datetime, timedelta
 
 import bottleneck
 import numpy as np
-import pandas as pd
 import xarray as xr
 
 warnings.filterwarnings("ignore")
 
 
 # PERFORMANCE METRICS
 def bias(sim, obs, dim="time"):
-    """Returns the bias between two time series.
+    r"""Return the bias between two time series.
```

Comment:
Why do you have some r""" here and not anywhere else:

Reply:
Because this docstring contains \ as part of the LaTeX code, so it is necessary for that but not for Others.